app/lib/efile/nj/nj1040_calculator.rb

```diff
@@ -280,7 +280,7 @@ def number_of_true_checkboxes(checkbox_array_for_line)
 
       def is_mfs_same_home
         is_mfs = @intake.filing_status_mfs?
-        is_same_home = @intake.tenant_same_home_spouse == 'yes'
+        is_same_home = @intake.tenant_same_home_spouse == 'yes' || @intake.homeowner_same_home_spouse == 'yes'
```

Comment:
[PEBBLE] Can use the generated helper methods `@intake.tenant_same_home_spouse_yes?` and `@intake.homeowner_same_home_spouse_yes?` to avoid typos with string literals.